app/models/manageiq/providers/azure/cloud_manager/vm_or_template_shared/scanning.rb

```diff
@@ -56,4 +56,18 @@ def has_proxy?
   def requires_storage_for_scan?
     false
   end
+
+  def require_snapshot_for_scan?
```

Comment:
@jerryk55 I thought you said that they suggested that both, managed and non-managed disks, always be snapshotted.